scripts/run_backend.sh

```diff
@@ -71,7 +71,9 @@ fi
 if docker image inspect "$image_name" >/dev/null 2>&1 ; then
     echo "Image '${image_name}' exists locally"
     echo "Starting $image_name"
-    docker run --network host --gpus all --name $1_backend --rm -it $image_name
+    ROS_URI="${ROS_MASTER_URI:-"localhost:11311"}"
```

Comment:
I would suggest we change this default value from "localhost:11311" to "http://localhost:11311" to conform with the actual ROS default string, then we can merge :-1:

Reply:
Agreed and changed.